iml-agent/src/daemon_plugins/snapshot.rs

```diff
@@ -121,7 +121,7 @@ impl DaemonPlugin for SnapshotList {
                                 lock.updated = true;
                             }
                         }
-                        delay_for(Duration::from_secs(60)).await;
+                        delay_for(Duration::from_secs(30)).await;
```

Comment:
```suggestion
                        delay_for(Duration::from_secs(10)).await;
```

Reply:
So, won't this be too low? With 2 fss with 20 snapshots each, this will leave 10 seconds out of 30 for mutating snapshot operations. This will mean statistically 2 of 3 operations will fail with `ldev.conf: file busy`

Reply:
I've checked and it isn't an issue for ldiskfs. Snapshots can be listed and created concurrently.